RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# How to have a release
+
+For Falter-releases, we usally do follow this workflow:
```

Comment:
Please change "we usually do follow this workflow" to "we follow this workflow".

Reply:
fixed in new version.